### ISSUE TITLE
[Bug] problems with screen reader (Jaws and NVDA)

### ISSUE THREAD

**Issue by mimosel** [gwdg/chat-ai#96]
### Describe the bug

Since the last update to version V0.9.0, some things are no longer possible with the screen reader (Jaws and NVDA).

The following problems exist:
1. The list of models can no longer be scrolled through using the arrow keys. The screen reader does not read out the models. When the arrow keys are pressed, sighted users can see the models being scrolled through.
The only workaround I can find is to search for the models using the search function (this always brings up one that is read aloud, but I cannot select it by pressing Return; instead, I have to select it with a simulated mouse click). If the search has more results, I am back to the problem above. The results are not read aloud.
A workaround would be to set which model should be selected at login in the settings. However, I am not sure if all of them are displayed to me.

2. I can no longer navigate to the upload button for uploading files. It is not displayed at all in the virtual view. If I know where it is, I can navigate there, but clicking the mouse does not trigger anything.

There are also a few unlabeled buttons, but the first two problems are the most pressing.

### Expected behavior

1. The list models should be scrollable by arrow keys. The models should be read out by the screen reader.
2. The upload button should be displayed in the virtual view.
All buttons should be labeled.

### Screenshots or logs

```shell

```

### Confirmation

- [x] I have confirmed this bug exists on the latest version
- [x] I have searched through existing issues
- [x] I have checked the Network connection
- [x] I reproduced the bug couple of times to ensure the source of that
